inst/extdata/sectoral/mappingClimateTrace.csv

```diff
@@ -0,0 +1,69 @@
+ClimateTrace;REMIND;Comment
+aluminum;Industry;
+bauxite-mining;Extraction;
+biological-treatment-of-solid-waste-and-biogenic;Waste;
+cement;Industry;Combustion and Process emissions not reported separately
+chemicals;Industry;Combustion and Process emissions not reported separately
+coal-mining;Energy|Supply|Fuels;
+copper-mining;Extraction;
+cropland-fires;Agriculture;
+crop-residues;Agriculture;
+domestic-aviation;w/o Bunkers|Energy|Demand|Transport;
+domestic-shipping;w/o Bunkers|Energy|Demand|Transport;
+domestic-wastewater-treatment-and-discharge;Waste;
+electricity-generation;Energy|Supply|Electricity;
+enteric-fermentation-cattle-operation;Agriculture;
+enteric-fermentation-cattle-pasture;Agriculture;
+enteric-fermentation-other;Agriculture;
+fluorinated-gases;F-Gases;processed separately
+food-beverage-tobacco;Industry;
+forest-land-clearing;Land-Use Change;
+forest-land-degradation;Land-Use Change;
+forest-land-fires;Land-Use Change;
+glass;Industry;
+heat-plants;Energy|Supply|Heat;
+incineration-and-open-burning-of-waste;Waste;
+industrial-wastewater-treatment-and-discharge;Waste;
+international-aviation;Transport|Pass|Aviation|International|Demand;
+international-shipping;Transport|Freight|International Shipping|Demand;
+iron-and-steel;Energy|Demand|Industry|Steel;
```

Comment:
this seems to be the only industry subsector that is split out explicitly. Is this still added to the industry total somewhere?